chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -397,7 +397,7 @@ impl PeerManagerActor {
             if bandwidth_used > REPORT_BANDWIDTH_THRESHOLD_BYTES
                 || total_msg_received_count > REPORT_BANDWIDTH_THRESHOLD_COUNT
             {
-                warn!(
+                debug!(
```

Comment:
also add 'target: "bandwith" - this way we'll be able to enable this log selectively

Reply:
done